x-pack/plugins/apm/server/lib/transactions/charts/get_anomaly_data/fetcher.ts

```diff
@@ -55,12 +55,8 @@ export async function anomalySeriesFetcher({
       size: 0,
       query: {
         bool: {
-          must: {
-            exists: {
-              field: 'bucket_span'
-            }
-          },
           filter: [
+            { exists: { field: 'bucket_span' } },
```

Comment:
`must` contributes to the score. `filter` does not contribute to the score and is cachable, therefore mostly faster.